Longhelp.md

```diff
@@ -24,13 +24,16 @@ usage: marathon_lb.py [-h] [--longhelp] [--marathon MARATHON [MARATHON ...]]
                       [--command COMMAND] [--sse] [--health-check]
                       [--lru-cache-capacity LRU_CACHE_CAPACITY]
                       [--haproxy-map] [--dont-bind-http-https]
-                      [--ssl-certs SSL_CERTS] [--skip-validation] [--dry]
+                      [--ssl-certs SSL_CERTS]
+                      [--marathon-ca-cert MARATHON_CA_CERT]
```

Comment:
@robsonpeixoto : It should have been rebased on top of latest master and I'm simply using the pre-commit hook as described in the README. 

Let me verify in a local branch here and ensure I actually had rebased on top of master..